### ISSUE TITLE
Deleted items still available in WebUI, changed items use old state/config

### ISSUE THREAD

**Issue by skupfer** [openhab/openhab-core#2247]
I successfully upgraded to OH3 from OH2. Today I added / changed / removed some items and especially the removed items from an ITEM.items file won't get removed from the WebUI. No error in log.

EDIT: Further information

I changed the item `Switch Shelly_483fda91aba0_Light "Light [%d]" { channel="shelly:shelly1:483fda91aba0:relay#output" }` to `Switch Shelly_483fda91aba0_Light "Light" { channel="shelly:shelly1:483fda91aba0:relay#output" }` and I still receive the following warning when using it: `2021-03-20 18:34:26.939 [WARN ] [e.internal.SseItemStatesEventBuilder] - Exception while formatting value 'OFF' of item Shelly_483fda91aba0_Light with format '%d': d != java.lang.String`

**Comment by Rossko57** [comment]
I suspect you would need to put [] in your Item file definition to erase the old 'state presentation pattern [%d] already in held in JSONDB, simply omitting any [presentation] may not be enough.

**Comment by lolodomo** [comment]
Items defined in a config file are read-only and they cannot be removed from MainUI.

Regarding the changed item, I imagine a simple restart of OH would have solved the problem.

I would suggest to close this issue.

**Comment by skupfer** [comment]
> Items defined in a config file are read-only and they cannot be removed from MainUI.
Yep. That's why I deleted them from the config file just as usual. Should have made this more clear.

> Regarding the changed item, I imagine a simple restart of OH would have solved the problem.
Best case scenario is that this shouldn't be necessary. Never had to restart before for simply deleting an item. That's why I guessed it's a bug or something and reported it here.

> I would suggest to close this issue.
Up to you. Since it got a label I am not sure whether I should close it now.
